ir/src/main/scala/org/scalajs/core/ir/Hashers.scala

```diff
@@ -96,11 +96,18 @@ object Hashers {
           mixBoolean(mutable)
           mixTree(rhs)
 
-        case ParamDef(ident, ptpe, mutable) =>
+        case ParamDef(ident, ptpe, mutable, rest) =>
           mixTag(TagParamDef)
           mixIdent(ident)
           mixType(ptpe)
           mixBoolean(mutable)
+          /* TODO Remove this test in the next major release.
+           * In 0.6.x we need this test so that the hash of a non-rest ParamDef
+           * emitted in 0.6.3 format is the same as an (implicitly non-rest)
+           * ParamDef emitted in 0.6.0 format.
+           */
+          if (rest)
+            mixBoolean(rest)
```

Comment:
Hmmm... spoke too soon maybe... It is strange that regenerating the has will not give you the same result...